plugins/metric/BetweennessCentrality.cpp

```diff
@@ -104,35 +129,11 @@ class BetweennessCentrality : public DoubleAlgorithm {
       stack<node> S;
       TLP_HASH_MAP<node, list<node>> P;
```

Comment:
After your changes, it seems that the **P** variable is no longer needed. There is no need to use it in both compute...() functions.

Reply:
I don't really understand your comment: the variable **P** is needed and used for the computation of the metric (with or without weight). See line 146. 
`for (auto v : P[w]) { ...
}`
Function `computeBFS` and `computeDikjstra` only compute the ancestors, number of paths and ordering for the shortest-paths from node s.

Reply:
Strange thing to not see what is under your nose. Forget my comment.